Cargo.toml

```diff
@@ -9,7 +9,7 @@ edition = "2021"
 uniswapx-rs = { path = "./crates/uniswapx-rs" }
 bindings-uniswapx = { path = "./crates/bindings-uniswapx" }
 
-artemis-core = { git = "https://github.com/paradigmxyz/artemis#70f3bce3488cf29f34197b9b2c0c02d329b95793"}
+artemis-core = { git = "https://github.com/paradigmxyz/artemis"}
```

Comment:
why not pin the commit here?

Reply:
Ah I was having some issues with cargo pinning a different version. Just reverted this!

Reply:
I pinned it because I was having some issues with the package resolution but have since fixed it.